templates/search.twig

```diff
@@ -0,0 +1,143 @@
+{% extends "base.twig" %}
+
+{% block content %}
+	<style>
+		#search-results-container {
+			width: 80%;
+			padding-top: 130px;
+			top:0;
+			bottom: 0;
+			left: 0;
+			right: 0;
+			margin: auto;
+		}
+		#search-results-container > h2 {
+			text-align: center;
+		}
+		#search-results-filter {
+			display: flex;
+			justify-content: center;
+		}
+		.filters-box {
+			padding-left: 20px;
+			border: 1px solid rgba(0,0,0,0.75);
+			-webkit-box-shadow: -4px 9px 5px 0 rgba(0,0,0,0.75);
+			-moz-box-shadow: -4px 9px 5px 0 rgba(0,0,0,0.75);
+			box-shadow: -4px 9px 5px 0 rgba(0,0,0,0.75);
+		}
+		.filters {
+			display: inline-block;
+			width: 30%;
+			float:left;
+		}
+		.results {
+			display: inline-block;
+			width: 70%;
+			float:left;
+		}
+		.results td {
+			vertical-align: top;
+		}
+		#results_settings {
+			float: right;
+		}
+		.result-post-date {
+			font-style: italic;
+			color: grey;
+		}
+	</style>
```

Comment:
Is there an issue associated with moving this into a stylesheet?

Reply:
@tcrsavage 
The styling I have added is temporary until our front end developer delivers the styling of this page, so this will be removed later on.